rollup.config.mjs

```diff
@@ -53,8 +54,9 @@ function buildConfig(format) {
         declaration: true,
         outDir: outDir,
       }),
+      json(),
```

Comment:
is this required for retrieving the sdk's version?

Reply:
yes